tsconfig.json

```diff
@@ -3,9 +3,10 @@
     "target": "ESNEXT",
     "module": "es2015",
     "moduleResolution": "node",
-    "jsx": "preserve",
     "noEmit": true,
     "strict": true,
+    "jsx": "react",
```

Comment:
should this not be preact?

Reply:
Nope, the way to tell TS that it's a Preact environment is to have:

```json
{
  "compilerOptions": {
    "jsx": "react",
    "jsxFactory": "h"
  }
}
```